app/src/main/java/de/danoeh/antennapod/adapter/DownloadlistAdapter.java

```diff
@@ -65,6 +65,7 @@ public View getView(int position, View convertView, ViewGroup parent) {
 
         holder.title.setText(request.getTitle());
         holder.secondaryActionIcon.setImageResource(ThemeUtils.getDrawableFromAttr(context, R.attr.navigation_cancel));
+        holder.secondaryActionIcon.setContentDescription(context.getString(R.string.cancel_download_label));
```

Comment:
Sorry, I just noticed something else. In other files, we set the content description to the whole `holder.secondaryActionButton`. Could you please do that, too?

Reply:
I've added the content description to `holder.secondaryActionButton`. But some of the checks are failing.